src/app/pages/tabs/profile/profile.page.ts

```diff
@@ -54,14 +56,23 @@ export class ProfilePage implements OnInit {
 
   async fetchProfileDetails() {
     var response = await this.profileService.profileDetails();
-    console.log(response);
-    this.formData.data = _.get(response, 'result');
-    if (this.formData.data.name) {
+    this.formData.data = _.get(response, 'user');
+    if (this.formData?.data?.about) {
       // TODO: remove the below line later
       this.showProfileDetails = true;
     }
   }
 
+  async doRefresh(event){
+    var res = await this.profileService.getProfileDetailsAPI();
+    this.formData.data = _.get(res, 'result');
+    this.localStorage.getLocalData(localKeys.USER_DETAILS).then(async (data)=>{
```

Comment:
can we write this logic in profile service only,,

Reply:
ok